src/Migrations/Version20190626090237.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use App\Migrations\AbstractTableNameMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20190626090237 extends AbstractTableNameMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->getTable(self::SPEAKER);
+        $table->dropColumn('interview_sent');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $table = $schema->getTable(self::SPEAKER);
+        $table->addColumn('interview_sent', 'boolean', ['default' => false]);
```

Comment:
There was no default value for this column, so when migrating down, it cause a desyncronized schema state with mapping.

Also, why not merging all migrations into one ?

To reproduce:

- migrate up and down,
- run `doctrine:schema:update --dump-sql`

Expected: Nothing to update

Currently:
```
The following SQL statements will be executed:
     ALTER TABLE speaker ALTER interview_sent DROP DEFAULT;
```

Reply:
Noted, I changed a lot on the migration part